### PR TITLE
Use hostmachine so that RPMs are in the right place (HOST dir)

### PR DESCRIPTION
**Pull request by mingwandroid**

### CODE REVIEW

**Comment by mingwandroid** [comment]
Fairly straightforward. I couldn't build a working xorg-x11-xauth-cos7-pcc64le without this.

File: conda_build/skeletons/rpm.py [hunk 1]
```diff
@@ -61,8 +61,8 @@
 BUILDSH = """\
 #!/bin/bash
 
-mkdir -p ${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot
-pushd ${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot > /dev/null 2>&1
+mkdir -p ${PREFIX}/{hostmachine}/sysroot
```
Comment (isuruf) [inline conda_build/skeletons/rpm.py]: Is it `host_machine`?

Comment (mingwandroid) [reply conda_build/skeletons/rpm.py]: No, but maybe it should be, but then we need to double escape things like `{{hostmachine}}` which looks weird, unless one of you python-istas have better ideas?

Comment (jjhelmus) [reply conda_build/skeletons/rpm.py]: {hostmachine} looks correct given line 520.  `host_machine` is only available in the cdt dictionary.

**Pull request merged**